app/views/errors/timeout.html.slim

```diff
@@ -0,0 +1,3 @@
+h1 For your security, we signed you out
+p="You have not done anything for #{Rails.configuration.x.user_timeout_minutes} minutes"
```

Comment:
Would setting this as a variable in the controller be a good idea?

Reply:
Or can we access it as a Devise class config var because it has been set there too

Reply:
The controller would not work because the app isn't loaded prior to the devise initializer being called. 

My first version had the constant defined on the `User` model, but I got constant (`User`) not defined on app start with that. I tried requiring the model and then got `ActiveRecord` not defined. 

Using `Rails.configuration.x` is the [solution suggested in the Rails guides](https://guides.rubyonrails.org/configuring.html#custom-configuration). 

To be honest, I do like having a single place to go to, to find app wide constants. It has worked well in previous projects. For example [here](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/main/config/application.rb).

Reply:
I do sometimes dislike the auto-requiring Rails does loading stuff. TBC I did mean something like:

```ruby
def timeout
  @duration = Rails.configuration.x.user_timeout_minutes
end
```

Only because I think it hints at what is going on in a view when in the controller layer and gives us a terser variable.

Reply:
I've updated the controller and view so that a variable is defined in the controller to hold the timeout, and that is then used in the view template.